### PR TITLE
Noetic support

### PR DESCRIPTION
**Pull request by chapulina**
Changes since Melodic:

* Only officially support Citadel onwards
* Don't need https://packages.osrfoundation.org
* This version will be officially released as upstream ROS packages

### CODE REVIEW

**Comment by chapulina** [comment]
Commented out `mav_msgs` because that's not yet available on Noetic: https://github.com/ethz-asl/mav_comm/issues/86

**Review by j-rivero — CHANGES_REQUESTED** [review]
I found some code still enabled for `Actuators`:

```
ros_ign_bridge/test/publishers/ign_publisher.cpp:160:  auto actuators_pub = node.Advertise<ignition::msgs::Actuators>("actuators");
ros_ign_bridge/test/publishers/ign_publisher.cpp:161:  ignition::msgs::Actuators actuators_msg;
ros_ign_bridge/test/publishers/ign_publisher.cpp:162:  ros_ign_bridge::testing::createTestMsg(actuators_msg);
ros_ign_bridge/test/publishers/ign_publisher.cpp:214:    actuators_pub.Publish(actuators_msg);

ros_ign_bridge/test/launch/test_ros_subscriber.launch:33:              <!-- /actuators@mav_msgs/Actuators@ignition.msgs.Actuators -->
ros_ign_bridge/test/launch/test_ign_subscriber.launch:28:              /actuators@mav_msgs/Actuators@ignition.msgs.Actuators

ros_ign_bridge/test/test_utils.h:1425:  void createTestMsg(ignition::msgs::Actuators &_msg)
ros_ign_bridge/test/test_utils.h:1442:  void compareTestMsg(const ignition::msgs::Actuators &_msg)
ros_ign_bridge/test/test_utils.h:1444:    ignition::msgs::Actuators expected_msg;
```
Do we want to keep it?

Also found references to other ROS releases or Ignition versions from Blueprint. I made a branch [`chapulina/noetic_jrivero` ](https://github.com/ignitionrobotics/ros_ign/compare/chapulina/noetic_jrivero?expand=1)

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,7 @@ sudo: required
 
 env:
   matrix:
-    - DOCKER_IMAGE=ubuntu:18.04 IGN_COLLECTION="blueprint"
-    - DOCKER_IMAGE=ubuntu:18.04 IGN_COLLECTION="citadel"
+    - DOCKER_IMAGE=ubuntu:20.04 IGNITION_VERSION="citadel" ROS_DISTRO="noetic"
```
Comment (j-rivero) [inline .travis.yml]: we might want to add the option of not defining a IGNITION_VERSION or just remove the one defined here (to simulate what's going to happen in ROS Buildfarm). I found problems with using that configuration on my local system defaulting to Blueprint versions.

```suggestion
    - DOCKER_IMAGE=ubuntu:20.04 ROS_DISTRO="noetic"
```

Comment (chapulina) [reply .travis.yml]: Removed

**Comment by chapulina** [comment]
> Do we want to keep it?

I left the Ignition parts since they're already working. They're not very useful without the ROS side though. I guess at least we can keep it from rotting. If you think it's best to comment it out I can do that.

> I made a branch 

Thanks, merged it in.

---

I also rebased the `noetic` branch onto `melodic` after #72 and #96 - the `noetic` branch was just matching `melodic` before anyways, with no commits of its own.

**Comment by j-rivero** [comment]
> I left the Ignition parts since they're already working. They're not very useful without the ROS side though. I guess at least we can keep it from rotting. If you think it's best to comment it out I can do that.

Nothing important, I thought on someone listing gz topics and found the Actuator in there. A part from a bit of compile/runtime overhead without real use.

**Review by j-rivero — APPROVED** [review]

**Pull request merged**